pom.xml

```diff
@@ -4,9 +4,65 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.venafi.vcert.sdk</groupId>
+    <groupId>io.github.venafi</groupId>
```

Comment:
The recomendation is, for project which are in github, the groupId should be io.github.username. More info [here](https://central.sonatype.org/publish/requirements/coordinates/)

Reply:
Yeah, I think `io.github.venafi` is what we need based on the Maven repository documentation since the other requires the ability to manage DNS records for the _venafi.com_ domain.  Will changing the `groupId` value here cause issues for any existing vcert-java consumers?

Reply:
Well, firstly the groupId was wrong because the goal of that is provide a global identifier which will be used to define a "namespace" for projects which are provided by individuals/organizations. For example in our case that will be `io.github.venafi` but currently the groupId is `com.venafi.vcert.sdk` which is containing the project's name also, when that is provided in the `artifactId` declaration. So given this error in the definition of the groupId then is clear that is needed to update it according to the specified in the documentation.
Related if that will impact negatively to the users, I think that will not happen because given that the groupId is mainly used to identify the artifacts in the maven repositories of binaries and until now we are not using any of  them to distribute the releases.